ghost/admin/app/services/feature.js

```diff
@@ -97,9 +119,40 @@ export default class FeatureService extends Service {
         }
     }
 
+    @computed()
+    get inAdminForward() {
```

Comment:
We can use this to toggle visibility of the Ember sidebar, loading of React components, etc. We could move it somewhere else as well if you prefer.